Makefile

```diff
@@ -28,6 +28,10 @@ CONTAINER_REGISTRY=gcr.io/dl-flow
 
 export DOCKER_BUILDKIT := 1
 
+.PHONY: clean-relic
+clean-relic:
+	rm -rf crypto/relic
+
 crypto/relic:
 	rm -rf crypto/relic
```

Comment:
```suggestion
crypto/relic: clean-relic
```

Reply:
I didn't want to do this, because i believe we removed the `PHONY` off `crypto/relic:` and if we have it depend of `clean-relic` that'd be the same as adding the `.PHONY` back on.

@jordanschalm @tarakby do you guys remember why the `.PHONY` was removed from `crypto/relic:`?

Reply:
I don't remember, but it might have been a mistake. An empty `crypto/relic` is already present on a fresh clone

Reply:
Not sure why it was removed, maybe because we didn't want to re-install Relic every time when we call `make install-tools` for instance? 
I think we should change `crypto/relic` and `crypto/relic/build` to `.PHONY` since we have actual folders called `relic` and `build` and this prevents the commands from running.

Reply:
I'm happy with them being `.PHONY`

Reply:
Agreed!

Reply:
just made `crypto/relic` `.PHONY`